todo/src/actions/actions.jsx

```diff
@@ -1,19 +1,6 @@
-export const INCREMENT = 'INCREMENT';
-export const DECREMENT = 'DECREMENT';
 
-// Our action creators will return
-// an action packet that our reducer will
-// receive. What does the action packet look like?
-// Note that the action creator is not at all
-// responsible for handling any of the actual
-// logic of updating the central Redux store. That
-// is left to the reducer(s).
-
-export const increment = () => ({
-  type:'INCREMENT'
+export const add = (payload) => ({
```

Comment:
Nice job on getting `add` reducer to work!
Possibly add one spacing character between the key/value pairs
```
type: 'ADD',
payload: payload
```